File: sdk/approute-sdk/golang/enum/transaction_status.go
package enum

// TransactionStatus represents the status of a purchase transaction.
type TransactionStatus string

const (
	TransactionInProgress         TransactionStatus = "in_progress"
	TransactionCompleted          TransactionStatus = "completed"
	TransactionPartiallyCompleted TransactionStatus = "partially_completed"
	TransactionCancelled          TransactionStatus = "cancelled"
)

File: sdk/approute-sdk/javascript/src/enums/transaction-status.ts
/**
 * Transaction lifecycle statuses.
 */
export const TransactionStatus = {
  IN_PROGRESS: "in_progress",
  COMPLETED: "completed",
  PARTIALLY_COMPLETED: "partially_completed",
  CANCELLED: "cancelled",
} as const;

export type TransactionStatus =
  (typeof TransactionStatus)[keyof typeof TransactionStatus];
